src/plugins/columnSorting/columnSorting.js

```diff
@@ -35,19 +43,44 @@ Hooks.getSingleton().register('afterColumnSort');
  * // as a object with initial order (sort ascending column at index 2)
  * columnSorting: {
  *  column: 2,
- *  sortOrder: true, // true = ascending, false = descending, undefined = original order
+ *  sortOrder: 'asc', // 'asc' = ascending, 'desc' = descending, 'none' = original order
  *  sortEmptyCells: true // true = the table sorts empty cells, false = the table moves all empty cells to the end of the table
  * }
  * ...
  * ```
- * @dependencies ObserveChanges
+ * @dependencies ObserveChanges moment
  */
 class ColumnSorting extends BasePlugin {
-
   constructor(hotInstance) {
     super(hotInstance);
-    this.sortIndicators = [];
+    this.sortIndicators = []; // TODO: It could be refactored, we are sorting just by one column, without saving state of previous sort indicators.
+    /**
+     * Visual index of last sorted column.
+     *
+     * @type {null|Number}
+     */
     this.lastSortedColumn = null;
```

Comment:
It looks like a cache for last sorted column called by mousedown in a header.

Reply:
For me it's ok.